### PR TITLE
Docs still rough

### PR DESCRIPTION
**Pull request by sjmackenzie**
Improve them

### CODE REVIEW

File: docs/vm_developer/README.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
-## Describe
-* build steps
-* debug tools of `SputnikVM`
-* how to run regression tests
-etc
+## Building SputnikVM
+```
+$ git clone git@github.com:ethereumproject/sputnikvm.git
+$ cd sputnikvm
+$ cargo build
```
Comment (sorpaas) [inline docs/vm_developer/README.md:7]: should be `cargo build --all`

File: docs/vm_integrator/README.md [hunk 2]
```diff
@@ -5,3 +5,56 @@
...
+            let value = M256::from_str(&client.get_storage_at(&format!("0x{:x}", address),
+                                                              &format!("0x{:x}", index),
+                                                              &block.number)).unwrap();
+            vm.commit_account(AccountCommitment::Storage {
+                address: address,
+                index: index,
+                value: value,
+            });
+        }
+        Err(err) => {
```
Comment (sorpaas) [inline docs/vm_integrator/README.md:49]: TODO: Update this when all RequireError is implemented.

**Comment by sorpaas** [comment]
Thanks for working on the docs!

File: docs/vm_integrator/README.md (hunk 2 above)
Comment (sjmackenzie) [reply docs/vm_integrator/README.md:49]: oh sure!

File: docs/vm_developer/README.md (hunk 1 above)
Comment (sjmackenzie) [reply docs/vm_developer/README.md:7]: thanks for the catch

**Pull request merged**